prow/entrypoint/run.go

```diff
@@ -138,8 +140,21 @@ func (o Options) ExecuteProcess() (int, error) {
 		arguments = o.Args[1:]
 	}
 	command := exec.Command(executable, arguments...)
-	command.Stderr = output
-	command.Stdout = output
+
+	stdout, err := command.StdoutPipe()
```

Comment:
The `CombinedOutput()` implementation does this:

```go
var b bytes.Buffer
c.Stdout = &b
c.Stderr = &b
```

Might be simpler?

Reply:
Ah okay, I wasn't aware of this. Thanks.